maven/impl-maven-embedded/src/main/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/pom/equipped/ConfigurationStageImpl.java

```diff
@@ -242,4 +243,16 @@ public ConfigurationDistributionStage setResumeFrom(String resumeFrom) {
         getInvocationRequest().setResumeFrom(resumeFrom);
         return this;
     }
+
+    @Override
+    public ConfigurationDistributionStage setAlternatePomFile(String pomFile) {
+        if(pomFile != null) {
+            File pom = new File(pomFile).getAbsoluteFile();
+            if (pom.isDirectory()) {
+                pom = new File(pom, "pom.xml");
```

Comment:
this should be a file - at least assuming from the method name

Reply:
This is as per the standard functionality offered by ` -f,--file <arg> :  Force the use of an alternate POM file (or directory with pom.xml).`

Reply:
Ok, leave it as it is